<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoStream</class>
 <widget class="QMainWindow" name="videoStream">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Stream</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="StreamView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10,1,0,1">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>RTSP URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputUrl">
        <property name="text">
         <string>https://bitmovin-a.akamaihd.net/content/playhouse-vr/m3u8s/105560.m3u8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>fast</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Speed">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>slow</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionScreenshot"/>
    <addaction name="actionRecord"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <widget class="QMenu" name="menuSpeed">
     <property name="title">
      <string>Speed</string>
     </property>
     <addaction name="actionFaster"/>
     <addaction name="actionFaster_2"/>
     <addaction name="actionNormal_Speed"/>
     <addaction name="actionSlower_fine"/>
     <addaction name="actionSlower"/>
    </widget>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop_Video"/>
    <addaction name="separator"/>
    <addaction name="menuSpeed"/>
   </widget>
   <addaction name="menuPlayback"/>
   <addaction name="menuSave"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionScreenshot">
   <property name="text">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop_Video">
   <property name="text">
    <string>Stop Video</string>
   </property>
  </action>
  <action name="actionFaster">
   <property name="text">
    <string>Faster</string>
   </property>
  </action>
  <action name="actionFaster_2">
   <property name="text">
    <string>Faster ( fine )</string>
   </property>
  </action>
  <action name="actionNormal_Speed">
   <property name="text">
    <string>Normal Speed</string>
   </property>
  </action>
  <action name="actionSlower_fine">
   <property name="text">
    <string>Slower( fine )</string>
   </property>
  </action>
  <action name="actionSlower">
   <property name="text">
    <string>Slower</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
